src/java.desktop/share/classes/javax/swing/ProgressMonitorInputStream.java

```diff
@@ -88,6 +89,8 @@ public ProgressMonitorInputStream(Component parentComponent,
         catch(IOException ioe) {
             size = 0;
         }
+        parentComponent = parentComponent;
+        message = message;
```

Comment:
These are both wrong I am sure. You want to assign to this.<var> don't you ? Not update the local var.

Reply:
Yes. Thanks for pointing out. Updated..